samples/sampler/stories/qa/Dropdown.js

```diff
@@ -1,88 +1,176 @@
 Dropdown.displayName = 'Dropdown';
 
-storiesOf('Dropdown', module)
-	.add(
-		'with 4 directions',
-		() => {
-			const itemCount = number('items', Config, {range: true, min: 0, max: 50}, 5);
-			const items = (new Array(itemCount)).fill().map((i, index) => `Option ${index + 1}`);
+const list = [
+	{children: 'hello 1', 'key': 'key1', 'aria-label': 'aria 1'},
+	{children: 'hello 2', 'key': 'key2', 'aria-label': 'aria 2', disabled: true},
+	{children: 'hello 3', 'key': 'key3', 'aria-label': 'aria 3'}
+];
 
-			return (
-				<>
-					<Dropdown
-						direction="left"
-						onSelect={action('onSelect')}
-						style={{position: 'absolute', top: 0, right: 0, width:'25%'}}
-						title="Left dropdown"
-					>
-						{items}
-					</Dropdown>
-					<Dropdown
-						direction="right"
-						onSelect={action('onSelect')}
-						style={{position: 'absolute', top: 0, left: 0, width:'25%'}}
-						title="Right dropdown"
-					>
-						{items}
-					</Dropdown>
-					<Dropdown
-						direction="down"
-						onSelect={action('onSelect')}
-						style={{position: 'absolute', top: 0, left: '50%', transform: 'translateX(-50%)', width:'25%'}}
-						title="Down dropdown"
-					>
-						{items}
-					</Dropdown>
-					<Dropdown
-						direction="up"
-						onSelect={action('onSelect')}
-						style={{position: 'absolute', top: ri.scaleToRem(440),  width:'25%'}}
-						title="Up dropdown"
-					>
-						{items}
-					</Dropdown>
-				</>
-			);
-		},
-		{
-			text: 'The basic Dropdown'
-		}
-	).add(
-		'Group in Scroller',
-		() => {
-			const itemCount = number('items', Config, {range: true, min: 0, max: 50}, 5);
-			const items = (new Array(itemCount)).fill().map((i, index) => `Option ${index + 1}`);
-			const dropdowns = [];
+class AutoDismissDropdown extends React.Component {
+	constructor (props) {
+		super(props);
+		this.state = {
+			open: true
+		};
+	}
+
+	handleClose = () => {
+		this.setState({open: false});
+	};
 
-			for (let i = 0; i < 30; i++) {
-				dropdowns.push({children: items, title: text('title', Config, 'Please select'), key: i});
-			}
+	render () {
+		return (
+			<div>
+				<Heading>Click in the blank area of the viewport to dismiss the Dropdown</Heading>
+				<Dropdown
+					onClose={this.handleClose}
+					open={this.state.open} // initial value is true
+				>
+					{['test1', 'test2', 'test3']}
+				</Dropdown>
+			</div>
+		);
+	}
+}
 
-			return (
-				<Scroller>
-					<Group
-						childComponent={Dropdown}
-						style={{position: 'absolute', top: 0, width:'50%'}}
-						onSelect={action('onSelect')}
-					>
-						{dropdowns}
-					</Group>
-				</Scroller>
-			);
-		},
-		{
-			text: 'The basic Dropdown'
-		}
+class DisabledDropdown extends React.Component {
+	constructor (props) {
+		super(props);
+		this.state = {
+			isDisabled: true
+		};
+	}
+
+	handleClick = () => {
+		this.setState({isDisabled: false});
+	};
+
+	render () {
+		return (
+			<div>
+				<Button onClick={this.handleClick}>enable dropdown</Button>
```

Comment:
This sample is very confusing. This button doesn't seem like work. Did you test?

Reply:
I took this sampler and the autoDismiss one from Sandstone and they behave the same there. I will fix here